files/fail2ban-abuseipdb.sh

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env bash
+#
+# Description:
+#   This script serves as a Fail2Ban `actionban` to report offending IP addresses to AbuseIPDB with custom COMMENTS.
+#   Mainly whenever Fail2Ban restarts, it calls the actionban function for each IP stored in the database file.
+#   If you restart your server often this script prevents duplicate reporting to AbuseIPDB by maintaining a local list of already reported IPs.
```

Comment:
The same thing can be done with:
```ini
norestored = true
```
See: https://github.com/fail2ban/fail2ban/blob/882e6d5e0011fe12bac8a356caea0394bc19b26c/ChangeLog?plain=1#L1023-L1028

Reply:
Thanks to point `norestored = true`, let me check.

Reply:
Ok, now the user can choose to rely on Fail2Ban's `norestored` setting using `BYPASS_FAIL2BAN` in `abuseipdb.local`. I have set it to `false` by default, ensuring that `actionban` will always trigger, allowing management at the script level.